### DISCUSSION TITLE
Code not rendered

### DISCUSSION THREAD

**Discussion by Eurekha-K** [community/community#65992] [Actions]
### Select Topic Area

Bug

### Body

Showing code not rendered while opening pdf files .Can anybody help  me with this error?
![Screenshot 2023-09-05 140917](https://github.com/community/community/assets/143890210/26d6219e-2e50-4591-b7d2-20305af81e4d)

**Comment by seyLu** [comment]
#### Hi @Eurekha-K 

Do you mean this?
![image](https://github.com/community/community/assets/98249191/3f18dbf0-e914-432c-bef8-1eaf6607d6f3)

If not, then probably a corrupted pdf file. Github should be able to load pdf files since 2015 (see [Github PDF Viewing Announcement](https://github.blog/2015-03-17-pdf-viewing/)).
